<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="windowTitle">
      <string>Packet Sniffer</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout">

        <!-- Control buttons layout -->
        <item>
          <layout class="QHBoxLayout" name="controlLayout">
            <item>
              <widget class="QPushButton" name="startButton">
                <property name="text">
                  <string>Start</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="stopButton">
                <property name="text">
                  <string>Stop</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="resumeButton">
                <property name="text">
                  <string>Continue</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>

        <!-- Packet table view -->
        <item>
          <widget class="QTableView" name="tableView">
            <property name="alternatingRowColors">
              <bool>true</bool>
            </property>
            <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
            </property>
          </widget>
        </item>

        <!-- Detail text area -->
        <item>
          <widget class="QPlainTextEdit" name="detailView">
            <property name="readOnly">
              <bool>true</bool>
            </property>
            <property name="placeholderText">
              <string>Выберите пакет для просмотра деталей...</string>
            </property>
          </widget>
        </item>

      </layout>
    </widget>
    <widget class="QStatusBar" name="statusbar"/>
  </widget>
  <resources/>
  <connections/>
</ui>
